src/components/Header/Header.js

```diff
@@ -11,7 +11,9 @@ const baseStyles = ({ theme }) => css`
   min-height: 64px;
   background-color: ${theme.colors.n900};
   padding: ${theme.spacings.mega};
-  ${theme.mq.mega`
+  position: sticky;
```

Comment:
Hate to be that guy, but IE11 doesn't support sticky: https://caniuse.com/#feat=css-sticky

I think you need to provide a fallback, unless we decide to not support it at all

Reply:
Why not just make it `position: fixed;`?

Reply:
Cause then you need to start handling the fixed header height as margin for the page

Reply:
Yes, we will need to add a margin on the container of every application using the `Header`. How do you guys think we should handle it for this component?